readme.md

```diff
@@ -44,6 +44,105 @@ This repository contains a project created in LabVIEW NXG preconfigured to build
 
 The following instructions can be used to add any generic web application created with any framework as a SystemLink plugin. A working plugin template is included in the `Web Server` folder called `webapp_plugin`. You can modify this template or duplicate the plugin by modifying the following configuration files. E.g. If you needed to create a second plugin (`webapp_plugin2`), you could duplicate the `webapp_plugin` directory, and make the following changes.
 
+### Plugins in SystemLink 2021R1 and newer
+
+Plugins in SystemLink 2021R1 appear in a navigation tree on the left of the page instead of the original homepage. In the config.json file, there is a section called `navigationItems` that specifies how the plugin appears in the navigation tree. Plugins can specify any number of items in the navigation tree. Each item should specify a unique id, a title token, and a URL beginning with a hash. Optionally, you can specify a group id and order within that group.
+
+**Note:** The Name, Access Control, and Icon sections only apply to plugins in earlier versions of SystemLink.
```

Comment:
This note was a little bit confusing to me because I didn't immediately realize that "sections" was referring to sections of this document rather than parts of the config.json file referenced in the paragraph above. Maybe we could say "sections of this document" instead of just "sections".

Also, when did the Access Control section become irrelevant? As far as I'm aware, we're not changing anything in 2021R1 related to this.

Reply:
I thought it became irrelevant with Org Modeling, but I was wrong. I've changed this to just say Name and Icon sections in the document are irrelevant.